utils/to/arn.go

```diff
@@ -72,6 +72,19 @@ func AwsRegionAccountFromContext(ctx context.Context) (*string, *string) {
 	return &region, &account
 }
 
+func AwsRegionAccountLambdaNameFromContext(ctx context.Context) (string, string, string) {
```

Comment:
"[if a function returns two or three parameters of the same type, or if the meaning of a result isn't clear from context, adding names may be useful](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)"

```suggestion
func AwsRegionAccountLambdaNameFromContext(ctx context.Context) (region, account, lambdaName string) {
```